docs/mockwbemserver.rst

```diff
@@ -1017,24 +1167,10 @@ Mock CIM repository
 User provider interfaces
 ------------------------
 
-.. automodule:: pywbem_mock._baseprovider
-
-.. autoclass:: pywbem_mock.BaseProvider
-   :members:
-
-   .. rubric:: Methods
-
-   .. autoautosummary:: pywbem_mock.BaseProvider
-      :methods:
-      :nosignatures:
-
-   .. rubric:: Attributes
-
-   .. autoautosummary:: pywbem_mock.BaseProvider
-      :attributes:
-
-   .. rubric:: Details
```

Comment:
Removing BaseProvider from the docs causes most methods of MethodProvider and InstanceWriteprovider not to have documentation, because they are inherited from BaseProvider and the links resolve only if BaseProvider is in the docs.

Reply:
Darn since that is just duplication.